core/src/test/java/io/questdb/test/griffin/O3SquashPartitionTest.java

```diff
@@ -786,8 +803,10 @@ private void testSquashPartitionsOnEmptyTable(String wal) throws Exception {
     private void testSquashPartitionsOnNonEmptyTable(String wal) throws Exception {
         assertMemoryLeak(() -> {
             // 4kb prefix split threshold
-            node1.getConfigurationOverrides().setPartitionO3SplitThreshold(4 * (1 << 10));
-            node1.getConfigurationOverrides().setO3PartitionSplitMaxCount(2);
+            Overrides overrides1 = node1.getConfigurationOverrides();
+            overrides1.setProperty(PropertyKey.CAIRO_O3_PARTITION_SPLIT_MIN_SIZE, 4 * (1 << 10));
+            Overrides overrides = node1.getConfigurationOverrides();
```

Comment:
nit - why are we getting `overrides` twice?

Reply:
That's just debris of auto inlining, I can simplify it